pkg/apis/legacysources/v1alpha1/sinkbinding_lifecycle.go

```diff
@@ -75,6 +75,8 @@ func (fbs *SinkBindingStatus) MarkBindingAvailable() {
 
 // Do implements psbinding.Bindable
 func (fb *SinkBinding) Do(ctx context.Context, ps *duckv1.WithPod) {
+	fb.MarkDeprecated(&fb.Status.Status, "SinkBindingDeprecated", "sinkbindings.sources.eventing.knative.dev are deprecated and will be removed in the future. Use sinkbindings.sources.knative.dev instead.")
```

Comment:
would you mind adding a UT that this actually works?

Reply:
ok

Reply:
Added.